src/Utilities/ToolTask.cs

```diff
@@ -1404,6 +1404,9 @@ public override bool Execute()
                         }
 
                         File.AppendAllText(_temporaryBatchFile, commandLineCommands, encoding);
+                        // Keep the encoding of standard output & error consistent with the console code page.
+                        StandardOutputEncoding = encoding;
+                        StandardErrorEncoding = encoding;
```

Comment:
Looks good - it just might be sfer to put this behind changewave (in case it breaks anybody)